tests/integration_tests/test_workflows_integration.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2022, Voltron Data.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+
+import pytest
+
+import benchalerts.workflows as flows
+from benchalerts.clients import GithubRepoClient
+
+
+@pytest.mark.parametrize("z_score_threshold", [None, 500])
+def test_update_github_status_based_on_regressions(
+    monkeypatch: pytest.MonkeyPatch, z_score_threshold
+):
+    """While this test is running, you can watch
+    https://github.com/conbench/benchalerts/pull/5 to see the statuses change!
```

Comment:
Here's a lil demo of this test (sped up)
![demo](https://user-images.githubusercontent.com/16600275/188706298-9b3c62eb-80ce-4b6c-96aa-54a5ea56594c.gif)

Reply:
Are you thinking some version of this for the icon? https://github.com/conbench/conbench/blob/main/conbench/static/bar-graph.png